manifests/repo.pp

```diff
@@ -159,6 +163,13 @@
         Apt::Source['zabbix'] -> Package<|tag == 'zabbix'|>
         Class['Apt::Update']  -> Package<|tag == 'zabbix'|>
       }
+      'windows' : {
+        if ($manage_choco == true) {
```

Comment:
this can be simplified:
```suggestion
        if $manage_choco {
```

Reply:
I have removed it and added it in the agent.pp